### PR TITLE
refactor: some improvements on the `feedback` command

### LINKED ISSUE #338: [FR]: Tiny final `kurtosis feedback` polishings
### Background & current behavior

Just a couple tiny things left with `kurtosis feedback`

PS: I created this issue using `kurtosis feedback --feature` 😄 

### Desired behavior

The helptext has a newline between the description and the `TIP`, so it's not as cramped.

`kurtosis feedback "I enjoy the enclave naming theme"` has backticks around it, to emphasize it's a command

`Github choose new issue type page` -> `Github "Choose New Issue" page`

`-f` is shorthand for `--feature`
`-b` is shorthand for `--bug`
`-d` is shorthand for `--docs`

Remove the `--github` flag (it's now redundant with the default behaviour with no flags)

### How important is this feature or improvement to you?

Nice to have, this feature would make using Kurtosis more enjoyable.

### LINKED ISSUE #371: [FR]: Remove the autofilled issue title text when filing new issues
### Background & current behavior

Because the [FR], [docs], etc. tags are added, it's possible to forget to add the title completely (because there is text in the title, Github will accept it).

### Desired behavior

We already have the labels automatically attached and GH allows filtering by labels, so remove the title text so that users are forced to fill out a title.

### How important is this feature or improvement to you?

Painful, the lack of this feature makes using Kurtosis frictionful.

### PR DESCRIPTION
**Pull request by leoporoli**
## Description:
Improvements on the feedback command
Fixes #338 
Fixes #371 

## Is this change user-facing?
YES (small changes, nothing to communicate from my perspective)

## References (if applicable):
#338 
#371

### CODE REVIEW

File: cli/cli/commands/feedback/feedback.go [hunk 1]
```diff
@@ -21,19 +21,20 @@ import (
 const (
 	commandShortDescription = "Give feedback"
 	commandDescription      = "Give feedback, file a bug report or feature request, or get help from the Kurtosis team. " +
-		"See below for the many ways you can get in touch with us.\n" +
+		"See below for the many ways you can get in touch with us.\n\n" +
 		"TIP: You can quickly type and send us feedback directly from the CLI. For example, " +
-		"kurtosis feedback \"I enjoy the enclave naming theme\" will open the Kurtosis Github " +
-		"choose new issue type page with the description pre-filled with \"I enjoy the enclave naming theme\"."
+		"`kurtosis feedback \"I enjoy the enclave naming theme\"` will open the Kurtosis Github " +
```
Comment (h4ck3rk3y) [inline cli/cli/commands/feedback/feedback.go]: s/kurtosis/Kurtosis/g

Comment (h4ck3rk3y) [reply cli/cli/commands/feedback/feedback.go]: s/Github/GitHub/g

File: docs/docs/reference/cli/feedback.md [hunk 2]
```diff
@@ -9,28 +9,27 @@ To quickly see all the ways you can get in touch with us, simply run:
...
+Additionally, the flags below can be used alone (will set the feedback type for GitHub which is the default feedback destination) and with the `--email` flag to specify the *type* of feedback you wish to provide:
+- The `--bug` flag can be used when you wish to submit a bug report to us. When this `--bug` flag is set without any destination flag, the CLI will take you directly to the [bug report issue creation page](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=&labels=bug&template=bug-report.yml&title=%5Bbug%5D%3A+) in our GitHub. When this `--bug` flag is set alongside the `--email` flag, the CLI will open an email draft with the subject pre-filled with: `[BUG]`, which will help our team triage and prioritize your report.
```
Comment (h4ck3rk3y) [inline docs/docs/reference/cli/feedback.md]: change the templates as they add the title

File: docs/docs/reference/cli/feedback.md [hunk 3]
```diff
@@ -9,28 +9,27 @@ To quickly see all the ways you can get in touch with us, simply run:
...
+- The `--feature` flag can be used when you wish to submit a feature request to us. When this `--feature` flag without any destination flag, the CLI will take you directly to the [feature request issue creation page](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=&labels=feature+request&template=feature-request.yml&title=%5BFR%5D%3A+) in our GitHub. When this `--feature` flag is set alongside the `--email` flag, the CLI will open an email draft with the subject pre-filled with: `[FEATURE_REQUEST]`, which will help our team triage and prioritize your request.
```
Comment (h4ck3rk3y) [inline docs/docs/reference/cli/feedback.md]: change the templates as they add the title

File: docs/docs/reference/cli/feedback.md [hunk 4]
```diff
@@ -9,28 +9,27 @@ To quickly see all the ways you can get in touch with us, simply run:
...
+- The `--docs` flag can be used when you wish to flag an issue with our documentation. When this `--docs` flag without any destination flag, the CLI will take you directly to the [docs issue creation page](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=leeederek&labels=docs&template=docs-issue.yml&title=%5BDocs%5D%3A+) in our GitHub. When this `--docs` flag is set alongside the `--email` flag, the CLI will open an email draft with the subject pre-filled with: `[DOCS]`, which will help our team triage and prioritize the issue.
```
Comment (h4ck3rk3y) [inline docs/docs/reference/cli/feedback.md]: change the templates as they add the title

File: docs/docs/reference/cli/feedback.md (hunk 2 above)
Comment (h4ck3rk3y) [reply docs/docs/reference/cli/feedback.md]: maybe put shorthands

**Review by h4ck3rk3y — APPROVED** [review]

File: docs/docs/reference/cli/feedback.md (hunk 2 above)
Comment (leoporoli) [reply docs/docs/reference/cli/feedback.md]: good catch!

**Pull request merged**